site/content/en/docs/administration/advanced/analytics.md

```diff
@@ -154,6 +155,38 @@ Client events:
 
 <!--lint enable maximum-line-length-->
 
+### Working time calculation
+
+Here is a short overview of how CVAT deals with the user's working time:
+
+- The user interface collects events when a user interacts with the interface
+(resizing canvas, drawing objects, clicking buttons, etc)
+The structure of one single event is described [here](#events-log-structure).
+
+- The user interface sends these events in bulks to the server.
+Currently, it uses the following triggers to send events:
+    - Periodical timer (~90 seconds)
+    - A user clicks the "Save" button on the annotation view
+    - A user opens the annotation view
+    - A user closes the annotation view (but not the tab/browser)
+
+- When events reach the server, it calculates working time based on timestamps of the events.
+
+- Working time is computed as a sum of time differences between
+the end of the previous event and the end of the next event.
+    - But if the difference is more then threshold (~100 seconds), the difference is not added to the working time
+    - Regardless different events may have `duration` field, we consider only `change:frame`
+    event as a continuous event when computing working time
+    - So, the only `duration` of the `change:frame` event is added to the working time.
+    Thresholding is not involved here.
```

Comment:
I find this layout confusing, because there are two components to the working time, but one of them is explained in the parent item, and one in the sub-item. I would suggest refactoring this to something like:

- The working time for an event is computed as the sum of the following:
  - The difference between the start time of the event and the end time of the previous event, if it is not more than 100 seconds.
  - The duration of the event, for events of type `change:frame`.